scripts/test-acceptance.sh

```diff
@@ -53,4 +53,4 @@ if [[ "${test_name}" = "accessibility" ]]; then
 fi
 
 echo "Running acceptance tests from ${test_name}.py against the sandbox..."
-nosetests ../acceptance/${test_name}.py --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
+nosetests ../acceptance/${test_name}.py:FullWorkflowRequiredTest --with-xunit --xunit-file=../acceptance/xunit-${test_name}.xml
```

Comment:
This is just to confirm the buggy tests pass on Jenkins without needing to wait on a full test run; will not be included in final merge.

Test plan:
- this subset of tests passes
- all tests pass
- PR reviewed and merged
- @yro (or I, whoever's free at that time) can continue with the ongoing release effort